### PR TITLE
Allows continuos recording of VMS data into HDF/CSV.

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

File: bin/VMSlogger [hunk 1]
```diff
@@ -53,62 +55,119 @@ parser.add_argument(
     action="store_true",
     help="save into HDF 5. Requires h5py (pip install h5py)",
 )
-parser.add_argument("--chunk-size", dest="chunk_size", default=300000, type=int)
+parser.add_argument("--chunk-size", dest="chunk_size", default=5000, type=int)
```
Comment (teweitsai) [inline bin/VMSlogger:58]: The `desp` here does not explain anything actually (the argument is `--chunk-size`).

File: bin/VMSlogger [hunk 2]
```diff
@@ -120,57 +179,95 @@ class Collector:
...
+                current_len = saved_len + len(self.cache)
+                logger.debug(
+                    f"Waiting {devices[self.index]}..{100 * (current_len)/self.size:.02f}% {current_len} of {self.size}"
+                )
+                if self.h5file is None:
+                    if current_len >= self.size:
+                        break
+                else:
+                    if self._save_hdf5():
+                        saved_len += self.chunk_size
```
Comment (teweitsai) [inline bin/VMSlogger:195]: Please consider to refactor the code to decrease the indentation if possible.

File: bin/VMSlogger [hunk 3]
```diff
@@ -120,57 +179,95 @@ class Collector:
...
+            self.cache.savetxt(self.filename, self.size, **kwargs)
+
+    async def collect_data(self):
+        try:
+            async with Domain() as domain:
+                remote = Remote(domain, "MTVMS", index=self.index + 1)
+                remote.tel_data.callback = lambda data: self.cache.newChunk(data, 0.001)
+                while True:
+                    self._create_file()
+                    await self.sample_file()
```
Comment (teweitsai) [inline bin/VMSlogger:246]: How to you stop this `while` loop? By `cancel`?

**Review by teweitsai — APPROVED** [review]
This PR looks good to me. Please consider to improve the documents. Actually, it is hard to say I really understand this PR. In addition, please consider to put the Dockerfile to the following common repository: https://github.com/lsst-ts/ts_Dockerfiles.

Thanks!

File: bin/VMSlogger (hunk 3 above)
Comment (pkubanek) [reply bin/VMSlogger:275]: Yes, by killing the process. But I added --single-shot option to quit after one file (with -s records) is recorded.

File: bin/VMSlogger (hunk 2 above)
Comment (pkubanek) [reply bin/VMSlogger]: Thanks, I considered splitting that before, finally made the split - see last comment.

File: bin/VMSlogger (hunk 1 above)
Comment (pkubanek) [reply bin/VMSlogger]: My fault, added. I also added epilog, providing more hints what this does.

**Pull request merged**